modules/nw-sctp-verifying.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * networking/using-sctp.adoc
+
+:image: registry.access.redhat.com/ubi8/ubi
+
+ifdef::openshift-origin[]
+:image: fedora:31
+endif::[]
+
+[id="nw-sctp-verifying_{context}"]
+= Verifying Stream Control Transmission Protocol (SCTP) is enabled
+
+You can verify that SCTP is working on a cluster by creating a Pod with an application that listens for SCTP traffic, associating it with a Service, and then connecting to the exposed service.
+
+.Prerequisites
+
+* Access to the Internet from the cluster to install the `lksctp-tools` package.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a Pod that runs an SCTP listener:
+
+.. Create a file named `sctp-server.yaml` that defines a Pod using the following YAML:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sctpserver
+  labels:
+    app: sctpserver
+spec:
+  containers:
+    - name: sctpserver
+      image: {image}
+      command: ["/bin/sh", "-c"]
+      args:
+        ["dnf install -y lksctp-tools && sctp_test -H localhost -P 30100 -l"]
+      ports:
+        - containerPort: 30100
+          name: sctpserver
+          protocol: SCTP
+----
+
+.. Create the Pod by entering the following command:
++
+----
+$ oc create -f sctp-server.yaml
+----
+
+. Create a Service for the SCTP listener Pod.
+
+.. Create a file named `sctp-service.yaml` that defines a Service using the following YAML:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: sctpserver
+  labels:
+    app: sctpserver
+spec:
+  selector:
+    app: sctpserver
+  ports:
+    - name: sctpserver
+      protocol: SCTP
+      port: 30100
+      targetPort: 30100
+  type: ClusterIP
+----
+
+.. To create the Service, enter the following command:
++
+----
+$ oc create -f sctp-service.yaml
+----
+
+. Create a deployment for the SCTP client.
+
+.. Create a file named `sctp-client.yaml` using the following YAML:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sctpclient
+  labels:
+    app: sctpclient
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sctpclient
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: sctpclient
+    spec:
+      restartPolicy: Always
+      containers:
+        - image: {image}
+          imagePullPolicy: IfNotPresent
+          name: sctpclient
+          command: ["sleep", "infinity"]
+----
+
+.. To create the Deployment object, enter the following command:
++
+----
+$ oc apply -f sctp-client.yaml
```

Comment:
Pod created from sct-client.yaml can not run sctp_test command:
[weliang@weliang FILE]$ oc rsh sctpclient-6ff49cbd4c-6nqfx
sh-4.4$ sctp_test
sh: sctp_test: command not found
sh-4.4$ exit